src/toolkit/matl_buy_policy.h

```diff
@@ -66,6 +66,7 @@ class MatlBuyPolicy : public Trader {
   /// @param manager the agent
   /// @param buf the resource buffer
   /// @param name a unique name identifying this policy
+  /// @param throughput a maximum on total transaction quantities
```

Comment:
Looks good, thanks!


On Apr 30, 2015, at 10:13 AM, Matthew Gidden <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/toolkit/matl_buy_policy.h<https://github.com/cyclus/cyclus/pull/1142#discussion_r29438173>:

> @@ -66,6 +66,7 @@ class MatlBuyPolicy : public Trader {
>    /// @param manager the agent
>    /// @param buf the resource buffer
>    /// @param name a unique name identifying this policy
> +  /// @param throughput a maximum on total transaction quantities


updated. let me know if they aren't satisfactory.
…<x-msg://92/#>
On Thu, Apr 30, 2015 at 10:06 AM, mbmcgarry <notifications@github.com<mailto:notifications@github.com>> wrote: In src/toolkit/matl_buy_policy.h <https://github.com/cyclus/cyclus/pull/1142#discussion_r29437434>: > @@ -66,6 +66,7 @@ class MatlBuyPolicy : public Trader { > /// @param<https://github.com/param> manager the agent > /// @param<https://github.com/param> buf the resource buffer > /// @param<https://github.com/param> name a unique name identifying this policy > + /// @param<https://github.com/param> throughput a maximum on total transaction quantities Could you expand a bit on this documentation to clarify the purpose of throughput? I needed to re-read the PR notes to understand the motivation of this addition. — Reply to this email directly or view it on GitHub <https://github.com/cyclus/cyclus/pull/1142/files#r29437434>.
-- Matthew Gidden, Ph.D. Postdoctoral Associate, Nuclear Engineering The University of Wisconsin -- Madison Ph. 225.892.3192

—
Reply to this email directly or view it on GitHub<https://github.com/cyclus/cyclus/pull/1142/files#r29438173>.

*************************************************************************
Meghan McGarry, PhD ~ http://mbmcgarry.github.io/
Postdoctoral Fellow, Consortium for Verification Technology
Computational Nuclear Engineering ~ http://cnerg.engr.wisc.edu
Engineering Physics ~ UW-Madison ~ 608-263-6974
*************************************************************************